src/mods.cpp

```diff
@@ -28,11 +28,44 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "util/strfnd.h"
 #include "convert_json.h"
 #include "exceptions.h"
+#include <vector>
+
+static void independentGetLine(std::istream& is, std::string& line)
+{
+    // The characters in the stream are read one-by-one using a std::streambuf.
```

Comment:
Oh my you don't want to google for this comment line. Seems this piece of code has been plagiarized all over the internet... Should we allow this in?

There are many issues with this function, including that it uses two references, and doesn't follow the code style. Maybe something in util/string.h already does the job? No idea...